libevmasm/AssemblyItem.cpp

```diff
@@ -323,12 +327,13 @@ string AssemblyItem::toAssemblyText(Assembly const& _assembly) const
 	return text;
 }
 
+// Note: This method is mainly used for debugging purposes.
```

Comment:
```suggestion
// Note: This method is exclusively used for debugging and testing purposes.
```

Reply:
Maybe actually even just ``debugging`` - even in the tests it's only printing error messages not actually testing stuff, isn't it?

Reply:
yes. I updated the comment.